exchanges/bybit/bybit_cfutures.go

```diff
@@ -0,0 +1,1255 @@
+	bybitFuturesAPIVersion = "/v2"
+
+	// public endpoint
+	cfuturesOrderbook          = "/public/orderBook/L2"
+	cfuturesKline              = "/public/kline/list"
+	cfuturesSymbolPriceTicker  = "/public/tickers"
+	cfuturesRecentTrades       = "/public/trading-records"
+	cfuturesSymbolInfo         = "/public/symbols"
+	cfuturesMarkPriceKline     = "/public/mark-price-kline"
+	cfuturesIndexKline         = "/public/index-price-kline"
+	cfuturesIndexPremiumKline  = "/public/premium-index-kline"
+	cfuturesOpenInterest       = "/public/open-interest"
+	cfuturesBigDeal            = "/public/big-deal"
+	cfuturesAccountRatio       = "/public/account-ratio"
+	cfuturesGetRiskLimit       = "/public/risk-limit/list"
+	cfuturesGetLastFundingRate = "/public/funding/prev-funding-rate"
+	cfuturesGetServerTime      = "/public/time"
+	cfuturesGetAnnouncement    = "/public/announcement"
+
+	// auth endpoint
+	cfuturesCreateOrder             = "/private/order/create"
+	cfuturesGetActiveOrders         = "/private/order/list"
+	cfuturesCancelActiveOrder       = "/private/order/cancel"
+	cfuturesCancelAllActiveOrders   = "/private/order/cancelAll"
+	cfuturesReplaceActiveOrder      = "/private/order/replace"
+	cfuturesGetActiveRealtimeOrders = "/private/order"
+
+	cfuturesCreateConditionalOrder       = "/private/stop-order/create"
+	cfuturesGetConditionalOrders         = "/private/stop-order/list"
+	cfuturesCancelConditionalOrder       = "/private/stop-order/cancel"
+	cfuturesCancelAllConditionalOrders   = "/private/stop-order/cancelAll"
+	cfuturesReplaceConditionalOrder      = "/private/stop-order/replace"
+	cfuturesGetConditionalRealtimeOrders = "/private/stop-order"
+
+	cfuturesPosition        = "/private/position/list"
+	cfuturesUpdateMargin    = "/private/position/change-position-margin"
+	cfuturesSetTrading      = "/private/position/trading-stop"
+	cfuturesSetLeverage     = "/private/position/leverage/save"
+	cfuturesGetTrades       = "/private/execution/list"
+	cfuturesGetClosedTrades = "/private/trade/closed-pnl/list"
+	cfuturesSwitchPosition  = "/private/tpsl/switch-mode"
+	cfuturesSwitchMargin    = "/private/position/switch-isolated"
+
+	cfuturesSetRiskLimit                   = "/private/position/risk-limit"
+	cfuturesGetMyLastFundingFee            = "/private/funding/prev-funding"
+	cfuturesPredictFundingRate             = "/private/funding/predicted-funding"
+	cfuturesGetAPIKeyInfo                  = "/private/account/api-key"
+	cfuturesGetLiquidityContributionPoints = "/private/account/lcp"
+
+	cfuturesGetWalletBalance           = "/private/wallet/balance"
+	cfuturesGetWalletFundRecords       = "/private/wallet/fund/records"
+	cfuturesGetWalletWithdrawalRecords = "/private/wallet/withdraw/list"
+	cfuturesGetAssetExchangeRecords    = "/private/exchange-order/list"
+)
+
+// GetFuturesOrderbook gets orderbook data for CoinMarginedFutures.
+func (by *Bybit) GetFuturesOrderbook(ctx context.Context, symbol currency.Pair) (Orderbook, error) {
+	var resp Orderbook
```

Comment:
consider using order.Item slice for this type.

Reply:
I am bit confused here. Can you please give more input.
Which of the following case or you meant something else?
1. var resp []order.Item
2. var resp Orderbook where Orderbook is of type`type Orderbook struct {
	Bids   []orderbook.Item
	Asks   []orderbook.Item
	Symbol string
	Time   time.Time
}`

Reply:
@Jdpurohit Sorry that was a typo; option number 2.